src/box/box.cc

```diff
@@ -782,28 +764,37 @@ box_check_election_timeout(void)
 	return d;
 }
 
-static void
-box_check_replication(void)
+static int
+box_check_uri_set_for_option(const char *option_name)
```

Comment:
Nit: let's name it simply `box_check_uri_set`.

Reply:
fixed